modules/imgproc/test/test_stackblur.cpp

```diff
@@ -310,6 +310,19 @@ TEST_P(StackBlur_GaussianBlur, compare)
     EXPECT_LE(cvtest::norm(dstS, dstG, NORM_INF), 13.);
 }
 
-INSTANTIATE_TEST_CASE_P(Imgproc, StackBlur_GaussianBlur, testing::Values(CV_8U, CV_16S, CV_16U, CV_32F));
+// Test very big stackblur kernel, even bigger than the input image size.
+// issue: https://github.com/opencv/opencv/issues/25498
+TEST_P(StackBlur_Type, bigKernel)
+{
+    Mat img_(346, 240, CV_8UC3);
+    Size ksize(741, 741);
+
+    const int dtype = get<0>(GetParam());
+    Mat img, dst;
+    convert(img_, img, dtype);
+    cv::stackBlur(img, dst, ksize);
```

Comment:
Please add some sanity check at least. I understand that you tried to reproduce crash, but after the crash fix the test is useless.

Reply:
Hi @asmorkalov, thanks for the code reviewing. I'm also wondering if we need to add such a test case.

Reply:
Removed the test code.